lib/components/term.js

```diff
@@ -299,11 +299,11 @@ export default class Term extends Component {
             height: '100%'
           }}
           /> :
-        <div
-          className={css('scrollbarShim')}
-          onMouseEnter={this.handleScrollEnter}
-          onMouseLeave={this.handleScrollLeave}
-          />
+            <div
```

Comment:
Ye, I would've probably used a couple of extra stateless components or a render function as well - should take a look at it some time. Thanks for the link.